addons/test_mail/tests/test_mail_activity.py

```diff
@@ -30,7 +30,93 @@ def setUpClass(cls):
 
 
 @tests.tagged('mail_activity')
-@tagged('at_install', '-post_install')  # LEGACY at_install
+class TestDoneActivities(TestActivityCommon):
+
+    @freeze_time("2025-09-15 10:00:00")
+    def test_search_on_date_done_includes_archived(self):
```

Comment:
To inline with other file naming, if we move to other file
```suggestion
    def test_mail_activity_mixin_search_done_and_archived((self):
```